.github/workflows/build-test-deploy.yml

```diff
@@ -8,9 +8,9 @@ on:
       - main
 
 env:
-  JAVA_VERSION: '17'
-  CONTAINER_REGISTRY_URL: 'ghcr.io/infonl'
-  APPLICATION_NAME: 'zaakafhandelcomponent'
+  JAVA_VERSION: "17"
```

Comment:
Why double quotes instead of single quotes? Is that a best practise?

Reply:
My editor force replaces it, trying to find the setting for it

I have this, still replaces
```
"yaml.format.singleQuote": true,
"prettier.singleQuote": true
```

Do you know?

Reply:
I did the editorconfig and was able to push a correctly quoted file 🥳